### ISSUE TITLE
Wrong datatype string

### ISSUE THREAD

**Issue by lukashenneke** [gnuradio/SigMF#266]
```
import numpy as np
import sigmf

data = np.zeros(1024, dtype=np.complex64)
print(sigmf.utils.get_data_type_str(data))
print(sigmf.__version__)
```

results in

```
cf64_le
1.0.0
```

Shouldn't it be cf32_le for np.complex64?

**Comment by jacobagilbert** [comment]
Thank you for the bug report.

This is a known issue with our v1.0 tag on this repository, it is fixed in the v1.0 on pip (`pip install sigmf` will get that version) and on master.

**Comment by lukashenneke** [comment]
Thanks.
Installing v1.1.0 from source fixed the issue.

FYI: I installed the package via `pip install sigmf`.
Source and Built Distribution on Pypi were uploaded on 27. Jan. 2022, I guess the bug fix is not included yet.
